essential-projects/iam_http
# Feature/use guest context if no token is provided

## What did you change?

Prevent a 401 error during resolving of the execution context, if the user provides no token.
Instead, use a guest-execution context.

## How can others test the changes?

Make a request against the route, without providing an auth token and see if you get an ExecutionContext for a guest.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
